.github/workflows/ci.yml

```diff
@@ -30,24 +30,20 @@ jobs:
         uses: actions/setup-python@v3
         with:
           python-version: "3.10"
-      - name: Get changed files
-        id: changed-files
-        run: |
-            if ${{ github.event_name == 'pull_request' }}; then
-                echo "changed_files=$(git diff --name-only -r HEAD^1 HEAD | xargs)" >> $GITHUB_OUTPUT
-            else
-                echo "changed_files=$(git diff --name-only ${{ github.event.before }} ${{ github.event.after }} | xargs)" >> $GITHUB_OUTPUT
-            fi
+      - name: Get modified apps
+        id: modified-apps-step
+        uses: ./.github/actions/modified_apps
       - name: Determine tests to run
         uses: actions/github-script@v7
         id: tests_to_run
         env:
-          CHANGED_FILES: ${{ steps.changed-files.outputs.changed_files }}
+          MODIFIED_APPS: ${{ steps.modified-apps-step.outputs.modified_apps }}
         with:
           script: |
-            const { CHANGED_FILES } = process.env;
+            const { MODIFIED_APPS } = process.env;
             const fs = require('fs');
             const path = require('path');
+            var has_tests = false;
```

Comment:
Should be called `has_python_tests` as the `snow app run` E2E is itself also a test. Can be confusing otherwise.